app/waf/src/controllers/adc.controller.ts

```diff
@@ -113,7 +116,7 @@ export class AdcController extends BaseController {
       userToken: await this.reqCxt.get(WafBindingKeys.Request.KeyUserToken),
       tenantId: await this.reqCxt.get(WafBindingKeys.Request.KeyTenantId),
     };
-    this.adcStCtr = new AdcStateCtrlr(adc, addonReq);
+    this.adcStCtr = new AdcStateCtrlr(adc, addonReq, this.reqCxt.name);
```

Comment:
Is it possible to combine `this.reqCxt.name` into `addonReq`? That means adding one more member to `AddonReqValues`, whose type is `string`, or `RequestContext` directly.